templates/repo/issue/milestones.tmpl

```diff
@@ -18,11 +18,11 @@
 				<div class="ui compact tiny menu">
 					<a class="item{{if not .IsShowClosed}} active{{end}}" href="{{.RepoLink}}/milestones?state=open&q={{$.Keyword}}">
 						{{svg "octicon-milestone" 16 "mr-3"}}
-						{{.i18n.Tr "repo.milestones.open_tab" .OpenCount}}
+						{{JsPrettyNumber .OpenCount}}&nbsp;{{.i18n.Tr "repo.issues.open_title"}}
```

Comment:
Here and below, I worry about i18n here, `Open` and `%d Open` may need different translations.

Reply:
Some languages like Arabic do `<open> <num>` but we can not indicate `num` as decimal because it's going to be pre-formatted as string. I guess I can move this to a generic `numopen` translation token with that string but it's probably going to require re-translation according to earlier comment.